gdpr/models.py

```diff
@@ -11,7 +12,64 @@
 
 class LegalReasonManager(models.Manager):
 
-    def create_from_purpose_slug(self, purpose_slug, source_object, issued_at=None, tag=None, related_objects=None):
+    def create_consent(self, purpose_slug, source_object, issued_at=None, tag=None, related_objects=None):
+        """
+        Create (or update, if it exist) a LegalReason with purpose slug for concrete object instance
+
+        Args:
+            purpose_slug: String of Legal Reason purpose
+            source_object: Source object this Legal Reason is related to
+            issued_at: When the Legal Reason consent was given
+            tag:
```

Comment:
Bez popisku?

Reply:
Bez popisku. Ty víš, co přesně ten tag znamená (pak ho tam klidně doplním)?

Reply:
Nevím. Ví @matllubos ?

Reply:
stále není doplněno

Reply:
Hmm, to se někde zatoulalo... :(